lib/ace/edit_session/bracket_match.js

```diff
@@ -37,17 +37,84 @@ var Range = require("../range").Range;
 
 function BracketMatch() {
 
+
+    this.findSurroundingBrackets = function(position) {
+    if (position.column == 0) return null;
```

Comment:
please format this code according to the commonly used style. Indent function body with 4 spaces, not less (like this) and not more (like lines 45-52). Left-align closing brace appropriately (e.g. at line 53 it should be 4 positions to the left).